Sources/ArcGISToolkit/Components/FeatureFormView/FeatureFormView.swift

```diff
@@ -15,9 +15,47 @@
 import ArcGIS
 import SwiftUI
 
-/// Forms allow users to edit information about GIS features.
+/// The `FeatureFormView` component enables users to edit field values of features
+/// in a layer using forms that have been configured externally, either in
+/// the Web Map Viewer or the Fields Maps Designer.
 ///
-/// - Since: 200.4
+/// Forms are currently only supported in maps. The form definition is stored
+/// in the web map itself and contains a title, description, and a list of "form elements".
+///
+/// `FeatureFormView` will support the display of form elements created by
+/// the Map Viewer or Field Maps Designer, including:
+///
+/// - Field Element - used to edit a single field of a feature with a specific "input type".
+/// - Group Element - used to group field elements together. Group Elements
+/// can be expanded, to show all enclosed field elements, or collapsed, hiding
+/// the field elements it contains.
+///
+/// A Field Element has a single input type object. The following are the supported
+/// input types and their associated toolkit view:
+///
+/// - Text Area - multi-line text area (TextInput)
+/// - Text Box - single-line text box (TextInput)
+/// - Combo Box - long lists of coded value domains (ComboBoxInput)
+/// - Radio Buttons - short lists of coded value domains (RadioButtonsInput)
+/// - Switch - two mutually exclusive values (SwitchInput)
+/// - Date/Time - date/time picker (DateTimeInput)
```

Comment:
Should these be alphabetized?

```suggestion
/// - Combo Box - long lists of coded value domains (ComboBoxInput)
/// - Date/Time - date/time picker (DateTimeInput)
/// - Radio Buttons - short lists of coded value domains (RadioButtonsInput)
/// - Switch - two mutually exclusive values (SwitchInput)
/// - Text Area - multi-line text area (TextInput)
/// - Text Box - single-line text box (TextInput)
```

Also, since these views aren't publicly exposed (at least for the time being), should we forego mentioning them by name here and instead just use this to list the supported input types?